packages/components/Checkbox/src/@types/index.d.ts

```diff
@@ -0,0 +1,5 @@
+declare module '*.svg' {
+  import { SvgProps } from 'react-native-svg';
+  const content: React.FC<SvgProps>;
+  export default content;
+}
```

Comment:
Did you add this yourself? I would be surprised if you had to manually add an index.d.ts file

Reply:
Yes. I added it manually, as per [FURN SVG Setup README](https://github.com/microsoft/fluentui-react-native/tree/master/apps/fluent-tester/src/FluentTester/TestComponents/Svg).

Reply:
Interesting. Thanks!